src/screens/Followings/FollowersScreen.js

```diff
@@ -11,12 +11,14 @@ import Search from '../DiscoveryScreenV2/elements/Search';
 import {getFollower} from '../../service/profile';
 import {Header} from '../../components';
 import {fonts} from '../../utils/fonts';
+import DiscoveryAction from '../../context/actions/discoveryAction';
 
 function FollowersScreen() {
   const route = useRoute();
   const {params} = route;
   const navigation = useNavigation();
   const [profileState, dispatchNavbar] = React.useContext(Context).profile;
+  const [discovery, discoveryDispatch] = React.useContext(Context).discovery;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `DiscoveryAction` and the use of `discoveryDispatch` from context are good additions. It's a good practice to manage state using context in React, especially when the state needs to be shared across multiple components. However, it would be better if you could validate the existence of `discoveryDispatch` before using it.
```diff
- const [discovery, discoveryDispatch] = React.useContext(Context).discovery;
+ const { discovery, discoveryDispatch } = React.useContext(Context)?.discovery || {};
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->